testsuite/forge-cli/src/main.rs

```diff
@@ -45,6 +43,18 @@ struct Args {
         default_value = "testnet-internal"
     )]
     helm_repo: String,
+    #[structopt(
+        long,
+        help = "The image tag currently is used for validators",
+        default_value = "devnet"
+    )]
+    image_tag: String,
+    #[structopt(
+        long,
+        help = "Image tag for validator software to do backward compatibility test",
+        default_value = "devnet"
+    )]
+    base_image_tag: String,
```

Comment:
we might want to separate these args into a subcommand for each test "suite", since they only make sense in the context of compat test. we can do that later though

Reply:
yea, u are right, I am working on a test dispatcher to indicate subcommand to run different test instead of this.